src/app/shared/myentries.service.ts

```diff
@@ -16,13 +16,13 @@
 import { OrgToolObject } from 'app/mytools/my-tool/my-tool.component';
 import { OrgWorkflowObject } from 'app/myworkflows/my-workflow/my-workflow.component';
 import { EntryType } from './enum/entry-type';
-import { DockstoreTool, Workflow } from './swagger';
+import { AppTool, DockstoreTool, Workflow } from './swagger';
 import { UrlResolverService } from './url-resolver.service';
 
 export abstract class MyEntriesService<E extends DockstoreTool | Workflow, O extends OrgToolObject<E> | OrgWorkflowObject<E>> {
   constructor(protected urlResolverService: UrlResolverService) {}
 
-  recomputeWhatEntryToSelect(entries: E[]): E | null {
+  recomputeWhatEntryToSelect(entries: Array<DockstoreTool | Workflow>): DockstoreTool | Workflow | null {
```

Comment:
Why did this have to change from `E`? And elsewhere in this file?

Reply:
I think I went through so many iterations I didn't realize this wasn't necessary anymore. Changed it back